code/controllers/subsystem.dm

```diff
@@ -196,6 +196,18 @@
 		if(SS_IDLE)
 			. = "  "
 
+/datum/controller/subsystem/proc/state_colour()
```

Comment:
minor thing. But this proc does not return the color. It returns the start of the font that has the color in it.